packages/backend/package.json

```diff
@@ -40,6 +40,7 @@
     "@backstage/plugin-azure-sites-backend": "workspace:^",
     "@backstage/plugin-badges-backend": "workspace:^",
     "@backstage/plugin-catalog-backend": "workspace:^",
+    "@backstage/plugin-catalog-backend-module-scaffolder-entities": "^0.0.0",
```

Comment:
`workspace:^`?

Reply:
Oh wut, how did I manage to get this to be a semver range

Fix'd